lib/utils.js

```diff
@@ -157,31 +159,38 @@ utils.createMapOfType = (obj, Type) => {
   if (!obj) return result;
 
   Object.keys(obj).forEach(key => {
+    // eslint-disable-next-line security/detect-object-injection
     result[key] = new Type(obj[key]);
   });
 
   return result;
 };
 
 utils.getMapKeyOfType = (obj, key, Type) => {
```

Comment:
I have some naming issues here, now this function is basically not only for getting a typed map, so name is no longer `getMapKeyOfType` and hiding it with `getMapKey` seems strange too me 🤔 
why not try renaming `getMapKeyOfType` to something more generic? with good jsdoc?

Reply:
I will try make it more generic :)

Reply:
Also in `162` line, without `eslint-disable`, we should cast key to String like `result[String(key)] = new Type(obj[String(key)]);`

Reply:
yeap, do the casting

did you make changes here? I don't think so